controllers/psyProfileController.ts

```diff
@@ -55,16 +49,29 @@ const updateValidators = [
     .customSanitizer(DOMPurify.sanitize)
     .isEmail()
     .withMessage('Vous devez spécifier un email valide.'),
-  check('address')
-    .trim()
-    .notEmpty()
-    .customSanitizer(DOMPurify.sanitize)
+  check('addresses')
+    .custom((values) => {
+      if (!Array.isArray(values) || values.length === 0) {
+        return false;
+      }
+      for (let i = 0; i < values.length; i++) {
```

Comment:
https://web.archive.org/web/20170403221045/https://blogs.oracle.com/greimer/entry/best_way_to_code_a

Reply:
Since the source you linked is quite old (2008) I find a benchmark website to test it now : https://jsben.ch/wY5fo 
Indeed, your version is better. And it could be even better if we save `values.length` in a const just before the loop :)